MapboxMaps.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
+	<true/>
+</dict>
+</plist>
```

Comment:
The entire MapboxMaps.xcworkspace should have been ignored. Let's check the `.gitignore` rules to figure out why it wasn't.

Reply:
looks like we ignore MapboxMaps.xcodeproj but not MapboxMaps.xcworkspace. Feel free to add that to the gitignore if you'd like.

Reply:
I wonder if this happened when I rigged [the navigation SDK’s CocoaPods integration test](https://github.com/mapbox/mapbox-navigation-ios/blob/main/Tests/CocoaPodsTest/PodInstall/Podfile/) to use the map SDK and Turf as development pods. I can remove this file in any case.